deploy_js_lambda.sh

```diff
@@ -0,0 +1,32 @@
+#/bin/bash
+
+STAGE=$1
+BUCKET=$2
```

Comment:
Could it be looked up by tags using the aws CLI? If not manual sounds like the best compromise.